src/AppInstallerCommonCore/ExperimentalFeature.cpp

```diff
@@ -56,6 +58,8 @@ namespace AppInstaller::Settings
             return ExperimentalFeature{ "Import Command", "import", "https://aka.ms/winget-settings", Feature::ExperimentalImport };
         case Feature::ExperimentalExport:
             return ExperimentalFeature{ "Export Command", "export", "https://aka.ms/winget-settings", Feature::ExperimentalExport };
+        case Feature::ExperimentalRestSource:
+            return ExperimentalFeature{ "Rest Source Support", "restSource", "https://aka.ms/winget-settings", Feature::ExperimentalRestSource };
```

Comment:
>https://aka.ms/winget-settings [](start = 78, length = 30)

Please remember to actually update the settings.md

Reply:
and the settings.schema.json

---
In reply to: [594659756](https://github.com/microsoft/winget-cli/pull/765#discussion_r594659756) [](ancestors = 594659756)

Reply:
Added to both settings.md and settings.schema.json.